### PR TITLE
fix(cli): tokens build crashed when run on result of tokens create

### LINKED ISSUE #2540: CLI `tokens build` command fails on tokens created with `tokens create`
### Version number

1.0.0-next.34
1.0.0-next.33

### Description of the bug

When building the created tokens, this happens:

```
Bulding tokens in ./design-tokens

🍱 Building typography
👷 theme - primary
👷 theme - secondary
file:///<example_dir>/node_modules/style-dictionary/lib/StyleDictionary.js:564
        throw new Error(err);
              ^

Error:
Reference Errors:
Some token references (93) could not be found.
```

### Steps To Reproduce

In an empty directory, run the following commands:
```
npx @digdir/designsystemet@1.0.0-next.34 tokens create \
   --accent "#007682" \
   --neutral "#003333" \
   --brand1 "#12404f" \
   --brand2 "#0054a6" \
   --brand3 "#942977" \
   --theme "theme" \
   --write
```

```
 npx @digdir/designsystemet@1.0.0-next.34 tokens build
```

See the crash.

### Additional Information

Also fails on 1.0.0-next.33

### PR DESCRIPTION
**Pull request by unekinn**
Closes #2540

### CODE REVIEW

File: packages/cli/src/tokens/design-tokens/default/semantic/style.json [hunk 1]
```diff
@@ -72,49 +72,17 @@
...
-        "$type": "typography",
-        "$value": {
-          "fontFamily": "{font.family}",
-          "fontWeight": "{font-weight.regular}",
-          "lineHeight": "{line-height.lg}",
-          "fontSize": "{font-size.5}",
-          "letterSpacing": "{letter-spacing.6}"
-        }
-      }
-    },
```
Comment (Barsnes) [inline packages/cli/src/tokens/design-tokens/default/semantic/style.json:116]: Is this supposed to be removed?

File: .changeset/strong-flowers-ring.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@digdir/designsystemet': minor
+---
+
+feat(cli): add --verbose option to "tokens build" for easier debugging
+fix(cli): update tokens template to reflect removal of `ingress` and renaming of `paragraph` to `body`
+fix(cli): `tokens build` crashed when run on result of `tokens create`
```
Comment (Barsnes) [inline .changeset/strong-flowers-ring.md:7]:
```suggestion
Fix CLI crashing when creating tokens:
- add --verbose option to "tokens build" for easier debugging
- update tokens template to reflect removal of `ingress` and renaming of `paragraph` to `body`
- `tokens build` crashed when run on result of `tokens create`
```

File: packages/cli/src/tokens/design-tokens/default/semantic/style.json (hunk 1 above)
Comment (unekinn) [reply packages/cli/src/tokens/design-tokens/default/semantic/style.json:116]: The github diff is a bit misleading without the context around it. It's just `ingress` as a whole which has been removed, while `body` still has `md`, `sm` and `xs` (plus the new `xl`). See:
image

File: .changeset/strong-flowers-ring.md (hunk 2 above)
Comment (unekinn) [reply .changeset/strong-flowers-ring.md:7]: Nice, I was uncertain of the structure here. I rewrote it somewhat, see newest commit

Comment (Barsnes) [reply .changeset/strong-flowers-ring.md]: Looks good, I'm unsure if changeset likes two lists - but lets merge this in and see!
We can always change it later if it comes out weird

**Review by Barsnes — APPROVED** [review]
🚀

**Pull request merged**